go/mysql/flavor_mysql.go

```diff
@@ -400,7 +402,8 @@ func (mysqlFlavor80) baseShowTablesWithSizes() string {
 }
 
 // supportsCapability is part of the Flavor interface.
-func (mysqlFlavor80) supportsCapability(serverVersion string, capability capabilities.FlavorCapability) (bool, error) {
+func (f mysqlFlavor80) supportsCapability(capability capabilities.FlavorCapability) (bool, error) {
+	serverVersion := f.serverVersion
```

Comment:
I'm guessing that if `f.serverVersion` is the zeroval, then this will just always return `false, nil.`, which seems fine to me. We should add a unit test case for that though.

Reply:
Added.